### PR TITLE
Add ruby/spec/language/block spec to the test suite

### PR DESCRIPTION
**Pull request by iliabylich**

### CODE REVIEW

File: lib/opal/nodes/def.rb [hunk 1]
```diff
@@ -199,7 +199,10 @@ def compile
               scope.mlhs_mapping[child] = tmp
 
               child.children.each do |child|
-                scope.add_temp variable(child.last)
+                # No support for nested mlhs yet
+                if child.type != :mlhs
+                  scope.add_temp variable(child.last)
+                end
```
Comment (elia) [inline lib/opal/nodes/def.rb]: shouldn't this blow up when it founds unspported args?

Comment (iliabylich) [inline lib/opal/nodes/def.rb]: What do you mean by unsupported arguments? From what I see in grammar.y `mlhs` can include only `norm_arg`, `restarg` or another `mlhs`. All other cases are rejected on the stage of translating to sexp.

Comment (elia) [inline lib/opal/nodes/def.rb]: what I meant is this:

```
raise NotImplementedError, "No support for nested mlhs yet" if child.type == :mlhs
scope.add_temp variable(child.last)
```

but maybe it doesn't make sense

Comment (IlyaBylich) [inline lib/opal/nodes/def.rb]: This code raises an error on the compilation stage and makes `ruby/spec/language/block_spec` unparsable. Could we just ignore nested `mlhs` arguments for now?

Comment (iliabylich) [inline lib/opal/nodes/def.rb]: This code raises an error on the compilation stage and makes ruby/spec/language/block_spec unparsable. Could we just ignore nested mlhs arguments for now?

Comment (elia) [inline lib/opal/nodes/def.rb]: :+1:

**Comment by elia** [comment]
Awesome

**Comment by iliabylich** [comment]
Thanks!

**Pull request merged**